src/MutableContainer.php

```diff
@@ -29,13 +30,19 @@ public function getItems(): array
         return $this->items;
     }
 
-    public function get(string $name): ItemInterface
+    public function get(string $identifier): ItemInterface
     {
-        return $this->items[$name];
+        return $this->items[$identifier];
     }
 
     public function reset(): void
     {
         $this->items = [];
     }
+
+    public static function getIdentifier(ItemInterface $item): string
```

Comment:
is this method going to be used/relevant outside of navigation package? if not, then this should be in some helper class, e.g. `ItemHelper`